pkgs/stdenv/linux/bootstrap-tools/scripts/unpack-bootstrap-tools.sh

```diff
@@ -9,6 +9,9 @@ echo Patching the bootstrap tools...
 if test -f $out/lib/ld.so.?; then
    # MIPS case
    LD_BINARY=$out/lib/ld.so.?
+elif test -f $out/lib/ld64.so.?; then
+   # ppc64(le)
+   LD_BINARY=$out/lib/ld64.so.?
```

Comment:
Does the musl unpack script need similar updating? Not sure what it's called in that situation.

Reply:
It certainly works without any additional changes, so I guess not …